javascript/testing.md

```diff
@@ -45,3 +45,47 @@ It:
 - Easy to read/understand
 - Catches bugs
 - Gives good coverage to effort ratio
+
+
+### Persisted nocks considered a antipattern
+
+We should avoid using persisted nocks so that we get a good idea of the request
+behaviour of our components. You might run into trouble when trying to do this
+especially when combining it with `nock.disableNetConnect();` and `nock.cleanAll();`
```

Comment:
Not sure about the wording here, because it sounds like we're placing `nock.disableNetConnect();` in the `afterEach` block, which wouldn't really help.

Maybe reversing them would work?

"when combining it with `nock.cleanAll(); in your `afterEach` block and `nock.disableNetConnect();` before running the test suite"

Reply:
Thanks! done!